content/ways-to-train/_assessment_only.html.erb

```diff
@@ -10,4 +10,8 @@
 
 <p>A number of universities and School Centred Initial Teacher Training programmes (SCITTs) around the country offer Assessment Only routes. Their criteria for entry may vary, and you will need to apply to them directly.</p>
 
-<p>For more information about Assessment Only options, you can view a list of <a href="/assessment-only-providers">approved providers in our directory</a>.
+<p>For more information about Assessment Only options, you can view a list of approved providers in our directory.
```

Comment:
While it's not _strictly_ necessary there should be a closing `</p>` tag here

Reply:
Done